src/components/Player/Livepeer/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Player } from "@livepeer/react";
+import {
+  LivepeerConfig,
+  createReactClient,
+  studioProvider,
+} from "@livepeer/react";
+
+export default function index() {
+  const client = createReactClient({
+    provider: studioProvider({
+      apiKey: "97f1716d-d1cd-4ebf-aa10-48d733b3f228",
+    }),
+  });
+
+  const livepeerTheme = {
+    colors: {
+      accent: "rgb(0, 145, 255)",
+      containerBorderColor: "rgba(0, 145, 255, 0.9)",
+    },
+    fonts: {
+      display: "Inter",
+    },
+  };
```

Comment:
Could these be static, outside the `index()` function? I think it'd work better not to create new clients on every re-render (I'm not a react pro tho, so not sure if this will actually happen)

Reply:
Yes, I have moved the client and theme out of the function.